src/test/fuzz/integer.cpp

```diff
@@ -95,6 +102,11 @@ void test_one_input(const std::vector<uint8_t>& buffer)
     (void)SipHashUint256(u64, u64, u256);
     (void)SipHashUint256Extra(u64, u64, u256, u32);
     (void)ToLower(ch);
+    (void)ToUpper(ch);
+    // ValueFromAmount(i) not defined when i == std::numeric_limits<int64_t>::min()
+    if (i64 != std::numeric_limits<int64_t>::min()) {
+        (void)ValueFromAmount(i64);
```

Comment:
Can test a roundtrip of `ValueFromAmount.getValStr<->ParseMoney`?

Reply:
Another very good idea! Thanks! Now added :)